packages/amplify-cli-core/src/feature-flags/featureFlags.ts

```diff
@@ -654,5 +654,13 @@ export class FeatureFlags {
         defaultValueForNewProjects: true,
       },
     ]);
+    this.registerFlag('category', [
```

Comment:
We need to change this based on our discussion earlier today on the structure for this. Needs to be per-category eg:

overrides: {
   auth: true/fals
}